site/app/controllers/HomePageController.php

```diff
@@ -90,8 +90,9 @@ public function getCourses($user_id = null, $as_instructor = false) {
             $user_id = $user->getId();
         }
 
-        $unarchived_courses = $this->core->getQueries()->getUnarchivedCoursesById($user_id);
-        $archived_courses = $this->core->getQueries()->getArchivedCoursesById($user_id);
```

Comment:
I believe you could remove these 2 functions that get called since this was the only place that they were used and you have created new functions rather than updating the old functions. No need to keep old, unused code.

Reply:
In terms of the OO Inheritance Model, I think there is value in keeping code that was essentially inherited from.